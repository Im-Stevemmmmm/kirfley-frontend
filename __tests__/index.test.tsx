import { assert } from 'console';

test('Example Test', () => {
  assert(true);
});
