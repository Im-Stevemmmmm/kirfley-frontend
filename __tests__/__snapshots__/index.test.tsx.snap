// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <title>
      Fern
    </title>
    <main>
      <div
        class="banner"
      >
        <div
          class="banner__child"
        >
          <h1
            class="banner__title"
          >
            Fern
            <span
              class="banner__logo"
            >
              <div
                style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
              >
                <div
                  style="box-sizing: border-box; display: block; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    role="presentation"
                    src="data:image/svg+xml;charset=utf-8,<svg width=\\"100\\" height=\\"75\\" xmlns=\\"http://www.w3.org/2000/svg\\" version=\\"1.1\\"/>"
                    style="max-width: 100%; display: block;"
                  />
                </div>
                <img
                  alt="Fern picture"
                  decoding="async"
                  src="/_next/image?url=%2Ffern.svg&w=320&q=75"
                  srcset="/_next/image?url=%2Ffern.svg&w=320&q=75 1x"
                  style="visibility: visible; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
              </div>
            </span>
          </h1>
        </div>
        <div
          class="banner__child banner__child___form"
        >
          <form
            class="form"
          >
            <h1
              class="form__title"
            >
              Log in
            </h1>
            <label
              class="form__label"
              for="usernameOrEmail"
            >
              Username or Email
            </label>
            <input
              class="form__input"
              id="usernameOrEmail"
              name="usernameOrEmail"
              placeholder="Username or email"
              type="text"
              value=""
            />
            <label
              class="form__label"
              for="password"
            >
              Password
            </label>
            <input
              class="form__input"
              id="password"
              name="password"
              placeholder="Password"
              type="password"
              value=""
            />
            <div
              class="buttonContainer"
            >
              <button
                class="form__button standard"
                type="submit"
              >
                Log In
              </button>
              <p>
                or
              </p>
              <button
                class="form__button standard"
                type="button"
              >
                Create an account
              </button>
            </div>
          </form>
        </div>
      </div>
      <div
        class="trendingContainer"
      >
        <h1
          class="trendingContainer__title"
        >
          The Latest Trends
        </h1>
        <h2>
          See what the world is talking about
        </h2>
        <div
          class="trendingContainer__trendingCards"
        >
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Stevemmmmm
            </h1>
            <p>
              Some text
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Someone
            </h1>
            <p>
              HIII XD
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Stevemmmmm
            </h1>
            <p>
              Some text
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Someone
            </h1>
            <p>
              HIII XD
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Stevemmmmm
            </h1>
            <p>
              Some text
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Someone
            </h1>
            <p>
              HIII XD
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Stevemmmmm
            </h1>
            <p>
              Some text
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Someone
            </h1>
            <p>
              HIII XD
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Stevemmmmm
            </h1>
            <p>
              Some text
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Someone
            </h1>
            <p>
              HIII XD
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Stevemmmmm
            </h1>
            <p>
              Some text
            </p>
          </div>
          <div
            class="trendingCard"
          >
            <h1
              class="trendingCard__title"
            >
              @Someone
            </h1>
            <p>
              HIII XD
            </p>
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;
